completion/available/nvm.completion.bash

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# nvm (Node Version Manager) completion
+
+if [ -r $NVM_DIR/bash_completion ]
```

Comment:
Do you also want to check for the presence of the `$NVM_DIR` variable first? What if it's not set?